src/main/java/org/carlmontrobotics/robotcode2023/subsystems/Drivetrain.java

```diff
@@ -120,6 +127,9 @@ public Drivetrain(Limelight lime) {
             for(CANSparkMax driveMotor : driveMotors) driveMotor.setSmartCurrentLimit(80);
         }
 
+        SmartDashboard.putNumber("testDistX", testDistX);
+        SmartDashboard.putNumber("testDistY", testDistY); 
+        SmartDashboard.putNumber("testRadTheta", testRadTheta);
```

Comment:
Move to a test command's initSendable such that you can actually modify them from smartdashboard.